### ISSUE TITLE
maven-site-plugin has broken dependencies

### ISSUE THREAD

**Issue by andreoss** [jcabi/jcabi-parent#100]
The plugin has incomplete (or broken) dependencies, and fails with (https://www.rultor.com/t/22557-656418158):
```
Caused by: java.lang.NoClassDefFoundError: org/apache/maven/doxia/sink/impl/XhtmlBaseSink
    at java.lang.ClassLoader.defineClass1 (Native Method)
    at java.lang.ClassLoader.defineClass (ClassLoader.java:756)
    at java.security.SecureClassLoader.defineClass (SecureClassLoader.java:142)
    at java.net.URLClassLoader.defineClass (URLClassLoader.java:468)
    at java.net.URLClassLoader.access$100 (URLClassLoader.java:74)
    at java.net.URLClassLoader$1.run (URLClassLoader.java:369)
    at java.net.URLClassLoader$1.run (URLClassLoader.java:363)
    at java.security.AccessController.doPrivileged (Native Method)
    at java.net.URLClassLoader.findClass (URLClassLoader.java:362)
    at org.codehaus.plexus.classworlds.realm.ClassRealm.loadClassFromSelf (ClassRealm.java:401)
```
Blocks: https://github.com/jcabi/jcabi-github/issues/1539 https://github.com/jcabi/jcabi-http/issues/203

**Comment by yegor256** [comment]
@andreoss can you provide a PR to fix this, please?

**Comment by andreoss** [comment]
@yegor256 Fixed in #96

**Comment by 0crat** [comment]
Job `gh:jcabi/jcabi-parent#100` is not assigned, can't get performer
